src/NuGet.Services.EndToEnd/Support/Clients/SearchService.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.Services.EndToEnd.Support
+{
+    /// <summary>
+    /// Represents a cloud search service with multiple instances.
+    /// </summary>
+    public class SearchService
```

Comment:
Because this class contains properties of a `SearchService`, and is not an actionable object, we should rename it to `SearchServiceProperties`. Also, the `QueryAsync` and similar methods in `V2V3SearchClient` should be moved back to using a `SearchServiceProperties` object.

Reply:
done